packages/next/src/server/lib/router-utils/setup-dev-bundler.ts

```diff
@@ -1041,6 +1041,18 @@ async function startWatcher(opts: SetupOpts) {
       )
     }
 
+    async function writeManifests(): Promise<void> {
+      await writeBuildManifest(opts.fsChecker.rewrites)
+      await writeAppBuildManifest()
+      await writePagesManifest()
+      await writeAppPathsManifest()
+      await writeMiddlewareManifest()
+      await writeActionManifest()
+      await writeFontManifest()
+      await writeLoadableManifest()
+      await writeFallbackBuildManifest()
```

Comment:
do we want to parallelize these? I have not idea if they have side effects so maybe not safe but seemingly the io parts could be done in parallel if they do not interact

Reply:
Overall we'll likely want to make these sync fs actually, but kept it as-is for now, can optimize later 👍